lib/UserInterface/textinput/StreamReaderWin.cpp

```diff
@@ -176,9 +176,20 @@ namespace textinput {
         return false;
       }
     } else {
+      //https://msdn.microsoft.com/en-us/library/windows/desktop/aa365690(v=vs.85).aspx
```

Comment:
Could you explain briefly what is the intent here, instead of pasting only a link.